.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,14 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Opencast Studio
+    url: https://github.com/elan-ev/opencast-studio/issues
+    about: Report issues with Opencast Studio in its own repository.
+  - name: Opencast Editor
```

We have still the old editor as default, should we point that (Until the new editor be the new default for the current and legacy releases).

```suggestion
  - name: Opencast Editor (New version)
```

---

I thought about that. The problem with “new version” is that users may not know which one is new and which one is not. You know that because you have seen the development, but imagine you are new to Opencast. You would have no idea. I also thought about adding as comment that it's the one not in the admin interface. But since it's configurable, even that is not universally true. That's why I ended up using what is the official name of the subproject, hoping that we get rid of the old one soon.

That said, I'm open for better suggestions because I definitely see the issue.

---

You're right! The new user don't know who editor is who.
Have an idea, can you make another step and show the UI of each? depending what ui they select you can redirect to the correct page.

Something like this:

```
Select how your editor looks like:

[Screenshot old editor] [Screenshot new editor]
```

---

Simpler approach. 
```suggestion
  about: Report issues with Opencast Editor in its own repository [If the UI looks like this](https://editor.opencast.org/)
```

If its troublesome is better left like originally you wrote, at the end we are going into that direction and if someone publishes an issue about the old editor, one can guide the user to the correct repo.

---

Would it be reasonable to list the version(s) the new one appeared in?  I would expect someone reporting a bug to know which Opencast version is in use...